src/main/kotlin/dev/merge/client/hris/models/EmployeeRequest.kt

```diff
@@ -0,0 +1,307 @@
+    @field:JsonProperty("personal_email")
+    val personalEmail: kotlin.String? = null,
+
+    /* The employee's mobile phone number. */
+    @field:JsonProperty("mobile_phone_number")
+    val mobilePhoneNumber: kotlin.String? = null,
+
+    /* Array of `Employment` IDs for this Employee. */
+    @field:JsonProperty("employments")
+    val employments: kotlin.collections.List<java.util.UUID>? = null,
+
+    @field:JsonProperty("home_location")
+    val homeLocation: java.util.UUID? = null,
+
+    @field:JsonProperty("work_location")
+    val workLocation: java.util.UUID? = null,
+
+    @field:JsonProperty("manager")
+    val manager: java.util.UUID? = null,
+
+    @field:JsonProperty("team")
+    val team: java.util.UUID? = null,
+
+    @field:JsonProperty("pay_group")
+    val payGroup: java.util.UUID? = null,
+
+    /* The employee's social security number. */
+    @field:JsonProperty("ssn")
+    val ssn: kotlin.String? = null,
+
+    /* The employee's gender. */
+    @field:JsonProperty("gender")
+    val gender: GenderEnum? = null,
+
+    /* The employee's ethnicity. */
+    @field:JsonProperty("ethnicity")
+    val ethnicity: EthnicityEnum? = null,
+
+    /* The employee's marital status. */
+    @field:JsonProperty("marital_status")
+    val maritalStatus: MaritalStatusEnum? = null,
+
+    /* The employee's date of birth. */
+    @field:JsonProperty("date_of_birth")
+    val dateOfBirth: java.time.OffsetDateTime? = null,
+
+    /* The date that the employee was hired, usually the day that an offer letter is signed. If an employee has multiple hire dates from previous employments, this represents the most recent hire date. Note: If you're looking for the employee's start date, refer to the start_date field. */
+    @field:JsonProperty("hire_date")
+    @Deprecated(message = "This property is deprecated.")
+    val hireDate: java.time.OffsetDateTime? = null,
+
+    /* The date that the employee started working. If an employee has multiple start dates from previous employments, this represents the most recent start date. */
+    @field:JsonProperty("start_date")
+    val startDate: java.time.OffsetDateTime? = null,
+
+    /* When the third party's employee was created. */
+    @field:JsonProperty("remote_created_at")
+    val remoteCreatedAt: java.time.OffsetDateTime? = null,
+
+    /* The employment status of the employee. */
+    @field:JsonProperty("employment_status")
+    val employmentStatus: EmploymentStatusEnum? = null,
+
+    /* The employee's termination date. */
+    @field:JsonProperty("termination_date")
+    val terminationDate: java.time.OffsetDateTime? = null,
+
+    /* The URL of the employee's avatar image. */
+    @field:JsonProperty("avatar")
+    val avatar: java.net.URI? = null,
+
+    /* Custom fields configured for a given model. */
+    @field:JsonProperty("custom_fields")
+    val customFields: kotlin.collections.Map<kotlin.String, kotlin.Any>? = null
+
+) {
+
+    data class Expanded(
```

Comment:
do we need to generate Expanded classes for *Request and *Response objects? from my understanding we should only need these + the normalize functions for the model class

Reply:
Sadly, it will be a lot more cost to do that due to openapi generator having an all or nothing sort of thing, let's talk offline about it but i'd prefer to leave this in (even tho they aren't useful)